webservices/resources/candidates.py

```diff
@@ -194,14 +176,12 @@ def get(self, **kwargs):
 
         return data
 
-    def get_candidate(self, args, page_num, per_page, candidate_id, committee_id):
```

Comment:
Ok, this is were the confusion was coming from, I thought `/committee/<id>/candidates` was changing. I understand what you are saying now. Thanks for explaining.